Add team norms to Cloud (#2055)
sourcegraph/about: handbook/engineering/cloud/index.md

By default, members of the team will provide a brief update about the goals they're working on (one update per iteration goal) — this will be a short version of the written update in the tracking issue.

## Team norms

### Code reviews

> - **Post-merge feedback:** It is important to make progress while getting feedback from other teammates in code reviews. On the one hand, the pull request author doesn't have to be blocked by all reviewers who the author intends to get feedback from; on the other hand, reviewers can still focus on their work on hands and leave feedback at their convenience. It is at pull request author's best judgement if a pull request should wait (for high-risk changes) or simply rely on post-merge feedback.

Reviewer: ```suggestion
- **Post-merge feedback:** It is important to make progress while getting feedback from other teammates in code reviews. On the one hand, the pull request author doesn't have to be blocked by all reviewers who the author intends to get feedback from; on the other hand, reviewers can still focus on their work on hands and leave feedback at their convenience. The pull request author should use their best judgement to decide if a pull request should wait (for high-risk changes) or simply rely on post-merge feedback.
```

Reviewer: I'm not quite sure what you mean by this:

```on the other hand, reviewers can still focus on their work on hands and leave feedback at their convenience```

Author: > I'm not quite sure what you mean by this:
> 
> `on the other hand, reviewers can still focus on their work on hands and leave feedback at their convenience`

Hmm, I was trying to say that everyone could do the work without spending all the time reviewing PRs (and worse, those PRs are blocking by their inputs).

Reviewer: Ok, that makes sense

> - **Approve to unblock:** When the reviewer thinks there is no obvious blocker and trusts the pull request author would take care of comments/questions/concerns (e.g. anwser to questions, explain rationale, act on code suggestions) before merging the pull request.

Reviewer: ```suggestion
- **Approve to unblock:** When the reviewer thinks there are no obvious blockers and trusts the pull request author will take care of comments/questions/concerns (e.g. answer to questions, explain rationale, act on code suggestions) before merging the pull request.
```
